fades/main.py

```diff
@@ -259,20 +263,20 @@ def go(argv):
 
     # store usage information
     usage_manager.store_usage_stat(venv_data, venvscache)
-
     if args.child_program is None:
         interactive = True
         l.debug("Calling the interactive Python interpreter")
-        p = subprocess.Popen([python_exe])
-
+        p = subprocess.Popen([python_exe, *python_options])
     else:
         interactive = False
         if args.executable:
             cmd = [os.path.join(venv_data['env_bin_path'], args.child_program)]
         else:
-            cmd = [python_exe, args.child_program]
+            cmd = [python_exe, *python_options, args.child_program]
         l.debug("Calling the child program %r with options %s",
                 args.child_program, args.child_options)
+
+        l.debug("Ejecutando %s ", cmd)
```

Comment:
This shouldn't be in Spanish!

OTOH, in the previous log line we're not logging the python-options, maybe you could add this field in that log line?